packages/components/carousel/__tests__/carousel.spec.ts

```diff
@@ -215,4 +215,26 @@ describe('Carousel', () => {
     expect(items[1].classList.contains('is-active')).toBeTruthy()
     done()
   })
+
+  it('async data', async () => {
+    const wrapper = _mount(
+      `
+        <div>
+          <el-carousel>
+            <el-carousel-item v-for="item in carousels" :key="item"></el-carousel-item>
+          </el-carousel>
+        </div>
+      `,
+      () => ({ carousels: [] })
+    )
+
+    await nextTick()
+    await wait(1000)
+    wrapper.setData({
+      carousels: [1, 2, 3],
+    })
+    await nextTick()
+    const items = wrapper.vm.$el.querySelectorAll('.el-carousel__item')
```

Comment:
**Nit** 

```ts
const items = wrapper.findAll('.el-carousel__item')
expect(items.at(0).classes('is-active')).toBe(true)
```

Reply:
Look like Vue Test Utils 2 do not have `at` method for wrapper-array, i use `items[x]` instead